docs/src/tutorials/get_component_data.md

```diff
@@ -55,3 +54,71 @@ bus1 = get_component(ACBus, sys, "nodeA")
 @show get_magnitude(bus1);
 nothing #hide
 ```
+
+# copied over from the System
```

Comment:
Maybe remove this?

Reply:
This tutorial is a to-do item and isn't currently visible to viewers. There are multiple comments in there that need to go, but I removed this one now.